recipe/meta.yaml

```
@@ -1,29 +1,30 @@
 {% set version = "24.2" %}
+{% set python_min = "3.8" %}
 
 package:
   name: packaging
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/p/packaging/packaging-{{ version }}.tar.gz
+  url: https://pypi.org/packages/source/p/packaging/packaging-{{ version }}.tar.gz
   sha256: c228a6dc5e932d346bc5739379109d49e8853dd8223571c7c5b55260edc0b97f
 
 build:
-  number: 0
+  number: 1
   noarch: python
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```

Isn't `--no-build-isolation` already specified (via `PIP_NO_BUILD_ISOLATION`) in the environment for all packages?

---

@henryiii good question, but I have no idea. Where do you think this being set?

---

Ah, I guess in `conda-build` itself: https://github.com/conda/conda-build/blob/0ad4bb6829b440caae9ae8a4db14ab1ff3788ab7/conda_build/build.py#L3020 ?

I'm now trying to think if there is a downside to doing this / why `grayskull` does this if it isn't needed at all.

---

PIP_NO_DEPENDENCIES is set too.

I'd guess someone didn't know this was already done.

---

Oh, seems that you're faster than me and opened https://github.com/conda/grayskull/issues/582 before I could open https://github.com/conda/grayskull/issues/583. :)

---

```suggestion
  script: {{ PYTHON }} -m pip install . -vv
```

It sounds like this might be the recommendation going forward.